conans/client/cache/cache.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, cache_folder):
         self._new_config = None
         self.editable_packages = EditablePackages(self.cache_folder)
         # paths
-        self._store_folder = self.config.storage_path or os.path.join(self.cache_folder, "data")
+        self._store_folder = os.path.join(self.cache_folder, "p")
```

Comment:
Is this intentional?
```suggestion
        self._store_folder = os.path.join(self.cache_folder, "data")
```

Reply:
Yes. New cache will not have any ``short_paths``, so better keep all folders to the minimum. The inner folder are already shortened to "e", "s", and the like.

Reply:
Thanks!